Formula/beanshell.rb

```diff
@@ -0,0 +1,23 @@
+class Beanshell < Formula
+  desc "Java Beanshell - Lightweight Scripting for Java"
+  homepage "http://www.beanshell.org/"
+  url "http://www.beanshell.org/bsh-2.0b4.jar"
```

Comment:
The latest version is `bsh-2.0b6`. Also, according to the homepage, upstream releases are now on Github so we should use the releases from https://github.com/beanshell/beanshell/releases.